pkg/ingester/ingester_test.go

```diff
@@ -6655,6 +6735,10 @@ func testIngesterCanEnableIngestAndQueryNativeHistograms(t *testing.T, sampleHis
 			Timestamp: 0,
 			Value:     1,
 		}},
+		Histograms: []model.SampleHistogramPair{{
+			Timestamp: 2,
```

Comment:
We don't expect the first histogram to make it in, because the flag is disabled and then:

```
// resend the histogram at time 2
	sampleHistograms[0].Timestamp = 2
```

Reply:
Oh I see